Functions/Admin/Register-RsPowerBI.ps1

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Licensed under the MIT License (MIT)
+
+
+function Register-RsPowerBI
+{
+    <#
+        .SYNOPSIS
+            This script configures Power BI integration.
+        
+        .DESCRIPTION
+            This script configures Power BI integration by providing details about a previously configured Azure Active Directory Web App
+        
+        .PARAMETER ClientId
+            Specify the Azure AD Client ID
+        
+        .PARAMETER ClientSecret
+            Client secret (client key) created to access the Azure AD App
+        
+        .PARAMETER AppObjectId
+            App Object ID representing the Azure AD App
+        
+        .PARAMETER TenantName
+            Azure AD Tenant Name ie. the organization name
+        
+        .PARAMETER TenantId
+            Azure AD Tenant ID
+        
+        .PARAMETER ResourceUrl
+            Default: 'https://analysis.windows.net/powerbi/api'
+            Overrides the default Power BI endpoint.
+        
+        .PARAMETER AuthUrl
+            Default: 'https://login.windows.net/common/oauth2/authorize'
+            Overrides the default Azure AD authentication URL.
+        
+        .PARAMETER TokenUrl
+            Default: 'https://login.microsoftonline.com/common/oauth2/token'
+            Overrides the default Azure AD token URL.
+        
+        .PARAMETER RedirectUrls
+            Redirect URLS - needs to match what is configured in the Azure AD app. General format:  http://{MyServerName}:80/reportserver/pages/LoginComplete.aspx
+        
+        .PARAMETER ReportServerInstance
+            Specify the name of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ReportServerVersion
+            Specify the version of the SQL Server Reporting Services Instance.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER ComputerName
+            The Report Server to target.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .PARAMETER Credential
+            The credentials with which to connect to the Report Server.
+            Use the "Connect-RsReportServer" function to set/update a default value.
+        
+        .EXAMPLE
+            PS C:\> Register-RsPowerBI -ClientId 'value1' -ClientSecret 'value2' -AppObjectId 'value3' -TenantName 'value4' -TenantId 'value5' -RedirectUrls 'value6'
```

Comment:
Removed it. This function had no example at all and habits like that just don't die easy :)